docs/notebooks/using_mclip.md

```diff
@@ -0,0 +1,188 @@
+---
+jupyter:
+  jupytext:
+    text_representation:
+      extension: .md
+      format_name: markdown
+      format_version: '1.3'
+      jupytext_version: 1.14.1
+  kernelspec:
+    display_name: Python 3 (ipykernel)
+    language: python
+    name: python3
+---
+
+# Multilingual Text To Image search with MultilingualCLIP
+
+
+Most text-image models are only able to provide embeddings for text in a single language, typically English. Multilingual CLIP models, however, are models that have been trained on multiple different languages. This allows the model the produce similar embeddings for the same sentence in multiple different languages.  
+
+This guide will show you how to finetune a multilingual CLIP model for a text to image retrieval in non-English languages.
+
+*Note, Check the runtime menu to me sure you are using a GPU./TPU instance, or this code will run very slowly.*
+
+
+
+## Install
+
+```python
+!pip install 'finetuner[full]'
+```
+
+## Task
+
+
+We'll be fine-tuning multilingual CLIP on the `toloka-fashion` dataset, which contains images and descriptions of fashion products in German.  
```

Comment:
if we are going to advertise them shouldn't we include a link at least? @bwanglzu @CatStark

Reply:
@LMMilliken switch back to XMarket